src/ert/config/queue_config.py

```diff
@@ -95,6 +95,13 @@ def create_local_copy(self) -> QueueConfig:
             self.queue_options,
         )
 
+    @property
+    def max_running(self) -> int:
+        for key, val in self.queue_options.get(self.queue_system, []):
+            if key == "MAX_RUNNING":
+                return int(val)
+        return 1
```

Comment:
Why 1? Docs says that zero is default and it will imply no limit (doc states that max_running is equal to number of realizations)

Reply:
https://github.com/equinor/ert/pull/6834

Reply:
I wasn't sure so I put a different number temporarily. 0 makes sense (maybe num CPUs for LOCAL driver?)